src/core/main.c

```diff
@@ -2419,10 +2416,10 @@ int main(int argc, char *argv[]) {
 finish:
         pager_close();
 
-        if (m)
+        if (m) {
                 arg_shutdown_watchdog = m->shutdown_watchdog;
-
-        m = manager_free(m);
+                m = manager_free(m);
+        }
```

Comment:
Dunno, the compiler can do this micro-opt for us too.

Reply:
it might semantically a bit more sense to me though, i.e. copy this out right before destroying it, if it's there...